File: 백준/Bronze/10807. 개수 세기/개수 세기.cc
#include<iostream>
using namespace std;

int main()
{
	int arr[101];
	int n, val, ans = 0; 
	cin >> n;
	for (int i = 1; i <= n; i++) cin >> arr[i];
	cin >> val;
	
	for (int i = 1; i <= n; i++) if (arr[i] == val) ans++;
	cout << ans;

	return 0;
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.util.Scanner;

class Main {
	public static void main(String[] args) throws Exception {
		Scanner scanner = new Scanner(System.in);
		int n = scanner.nextInt();
		int[] arr = new int[n];
		
		for(int i = 0; i<n; i++) 
			arr[i] = scanner.nextInt();
		
		int v = scanner.nextInt();
		int ans = 0;
		for(int i = 0; i<n; i++) 
			if(arr[i] == v) ans++;
		
		System.out.println(ans);
	}
}